src/flask_container_scaffold/app_scaffold.py

```diff
@@ -2,76 +2,99 @@
+        on a Flask app. These can be specified in the following order, each
+        overriding the last, if specified:
+        - via config mapping
+        - via Flask settings.cfg file
+        - via environment variable 'FLASK_SETTINGS'
+        """
+        config_not_loaded = True
+        if self.config is not None:
             # load the config if passed in
             self.app.config.from_mapping(self.config)
-            # Next, load from override file, if specified
-        if os.environ.get('SETTINGS'):
-            self.app.config.from_envvar('SETTINGS')
-        # Next, load env vars directly, which will override
-        # any previous settings.
-        self.setup_core_config_from_env()
-        # Loop again, making sure everything in env_list is set somehow
-        # on the config object.
-        self.setup_core_config_from_env(verify=True)
-        # Load the defaults yaml file
-        default_conf = (self.app.config.get('DEFAULTS_FILE') or False)
-        if default_conf:
-            self.app.config.update(
-                {'default_params': load_yaml(default_conf)})
+            config_not_loaded = False
+        # load the instance config, if it exists and/or is required
+        try:
+            self.app.config.from_pyfile('settings.cfg', silent=self.silent)
+            config_not_loaded = False
+        except:
+            config_not_loaded = True
+            pass
+        # Do any additional config specified in the FLASK_SETTINGS file,
+        # if it exists. We only want to fail in the case where settings are
+        # required by the app.
+        if ((config_not_loaded and self.silent == False)
+            or os.environ.get('FLASK_SETTINGS')):
+            self.app.config.from_envvar('FLASK_SETTINGS')
 
-    def get_config_list(self):
-        # TODO: pull this from a file or take list instead
-        config_list = ['GIT_BASE_URL', 'DEFAULTS_FILE']
-        return config_list
 
-    def setup_core_config_from_env(self, verify=False):
-        # loop through the list, setting each value from env if it exists
-        for i in self.get_config_list():
-            if os.environ.get(i):
-                self.app.config[i] = os.environ.get(i)
-            if verify and not self.app.config.get(i):
-                raise ValueError(f"No {i} set for Flask application")
+    def _load_custom_settings(self):
+        """
+        Load any custom configuration for the app from:
+        - app.config['CUSTOM_SETTINGS']
+        - environment variable 'CUSTOM_SETTINGS'
+        """
+        configurator = AppConfigurator(self.app, self.relative)
+        if self.app.config.get('CUSTOM_SETTINGS') is not None:
+            # load the config if passed in
+            custom = self.app.config.get('CUSTOM_SETTINGS')
+            configurator.parse(custom)
+        # Next, load from override file, if specified
+        if os.environ.get('CUSTOM_SETTINGS'):
```

Comment:
This is a nit but I notice on line 93 "is not None" is explicitly stated in the comparision while it's not here. It seems like the two may as well be checked the same way?